nx.json

```diff
@@ -36,7 +36,8 @@
     "default": [
       "{projectRoot}/**/*",
       "globalBuildAffectingConfig",
-      "globalNonBuildAffectingConfig"
+      "globalNonBuildAffectingConfig",
+      "!{workspaceRoot}/**/*"
```

Comment:
how does this remove `docs` specifically? According to the nx docs:

> {workspaceRoot} is a key word that is replaced by the root path of your workspace.

so `workspaceRoot` is our root directory - the one above `packages`, right? So that glob pattern would exclude .. everything 😅 ?

Reply:
ackkk my mistake - sorry I lost touch on this one - fixing now